src/ImageSharp/Common/Helpers/Numerics.cs

```diff
@@ -61,6 +60,12 @@ public static int LeastCommonMultiple(int a, int b)
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
     public static nint Modulo4(nint x) => x & 3;
 
+    /// <summary>
+    /// Calculates <paramref name="x"/> % 4
+    /// </summary>
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static nuint Modulo4(nuint x) => x & 3;
```

Comment:
`x % 4` should already get optimized to `x & 3` by the JIT since `x` is `nuint` (and therefore definitely unsigned)

Reply:
Thanks yeah, I just kept it the same for consistency.